site/content/contribute/server/developer-setup/centos.md

```diff
@@ -33,14 +33,14 @@
     sudo chmod +x /usr/local/bin/docker-compose
     ```
 
-3. Install Go:
+3. Install Go (modify installation to latest Go version from https://golang.org/dl/):
 
     ```sh
     sudo yum group install "Development Tools"
     sudo yum install -y wget libpng12
     sudo rm -rf /usr/local/go
-    wget https://storage.googleapis.com/golang/go1.12.linux-amd64.tar.gz
-    sudo tar -C /usr/local -xzf go1.12.linux-amd64.tar.gz
+    wget https://dl.google.com/go/go1.13.2.src.tar.gz
```

Comment:
maybe update to 1.13.3 since was released in the same day as well and a ben point out

Reply:
Haha strange. Will update.